### PR TITLE
coverity: Use GCC 4.9.

### PR DESCRIPTION
**Pull request by fwalch**
@justinmk 

For whatever reason, the coverity job does not install the APT packages.. `Adding APT Sources (BETA)` is missing from the build log.

### CODE REVIEW

**Comment by fwalch** [comment]
Adding `os: linux` seems to fix it.. looks like a bug.

File: .travis.yml [hunk 1]
```diff
@@ -47,23 +50,22 @@ addons:
...
     packages:
-      # Basic Neovim/test dependencies.
       - autoconf
       - automake
       - build-essential
+      - bzr-fastimport
+      - clang-3.6
       - cmake
+      - g++-4.9
+      - g++-5
```
Comment (justinmk) [inline .travis.yml]: @fwalch Just curious, why are these needed?

**Comment by justinmk** [comment]
Thank you!

File: .travis.yml (hunk 1 above)
Comment (fwalch) [inline .travis.yml]: I think it was for compiling some third-party dependency.
